ledger-core/rms/proto_records.proto

```diff
@@ -120,26 +124,103 @@ message RLineMemoryExpected {
 message RLineMemoryReuse {
     option (insproto.id) = 26;
     RecordBody RecordBody = 19 [(gogoproto.embed)=true];
+
+    Reference PrevRef = 24;
+    Reference RootRef = 25;
 }
 
 message RLineInboundRequest {
     option (insproto.id) = 27;
     RecordBody RecordBody = 19 [(gogoproto.embed)=true];
+
+    Reference PrevRef = 24;
+    Reference RootRef = 25;
+
+    CallType CallType = 40;
+    uint32 CallFlags = 41 [(gogoproto.casttype) = "CallFlags"];
+    uint32 CallAsOf  = 42 [(gogoproto.casttype) = "PulseNumber"];
+    Reference Caller = 43;
+    Reference Callee = 44;
+
+    Reference CallSiteDeclaration = 45;
+    string CallSiteMethod = 46;
+
+    uint32 CallSequence = 47;
+    Reference CallReason = 48;
+    Reference RootTX = 49;
+    Reference CallTX = 50;
+
+    Reference ExpenseCenter = 51;
+    Reference ResourceCenter = 52;
+
+    CallDelegationToken DelegationSpec = 54;
```

Comment:
not in the record (only in a message) as well as ProducerSignature, RegistrarSignature, RegistrarDelegationSpec, CallRequestFlags, KnownCalleeIncoming, CallOutgoing

Reply:
fixed